src/Button/BorderlessButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {
+  FunctionComponent, ReactElement, useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { darken } from 'polished';
+import { BaseTheme } from '../Theme';
+import { StyledButton } from './StyledButton';
+import { ButtonProps } from './ButtonProps';
+
+const StyledPrimaryButton = styled(StyledButton)`
+  background: ${({ theme }): string => (theme.color.background.info.light)};
```

Comment:
I think the idea behind the borderless button was that it would appear on screen as just the label text, e,g, for those cases where we have an edit/delete icon in a table. In that case, I think we'd want to have a `transparent` background instead, and then provide some other visual indication for the `:hover` and `:active` states.

Reply:
Thanks for your help! Fixed through 769e9fd